edx_django_utils/monitoring/internal/backends.py

```diff
@@ -165,6 +178,11 @@ def tag_root_span_with_error(self, exception):
         root_span = self.dd_tracer.current_root_span()
         root_span.set_exc_info(type(exception), exception, exception.__traceback__)
 
+    def set_local_root_span_name(self, name, group=None, priority=None):
+        # For Datadog, this updates the 'operation_name' to the given name.
```

Comment:
Does it not also update the resource name? I think that's the main thing we want to change, here. (NR does not do a good job of documenting what the "group" is, but I think it's actually mostly similar to DD's operation_name.)

Reply:
In my testing, it didn't, and I couldn't find anything that would let me set it.